src/main/features/claim/routes/pay.ts

```diff
@@ -165,6 +184,12 @@ export default express.Router()
 
       await new DraftService().save(draft, user.bearerToken)
 
+      trackCustomEvent(`Post payment for externalId: ${externalId}`,{
```

Comment:
Same here, I think there's only value in tracking the failure

Reply:
As above